product-service/serverless.yml

```diff
@@ -1,22 +1,28 @@
 frameworkVersion: '2'
 service: ducati-product-service
 
-custom:
-  webpack:
-    webpackConfig: webpack.config.js
-    includeModules: true
-
 plugins:
+  - serverless-bundle
+  - serverless-dotenv-plugin
```

Comment:
I think serverless-dotenv-plugin is unnecessary, because variables engine is already active in Serverless 2.26.0+:
https://github.com/neverendingqs/serverless-dotenv-plugin/discussions/155

I just used built-in env
https://www.serverless.com/framework/docs/environment-variables

but it's just note.